db/glue.c

```diff
@@ -6224,6 +6224,10 @@ int comdb2_is_user_op(char *user, char *password)
     return rc;
 }
 
+int comdb2_iam_master() {
+    return (!thedb || thedb->master == gbl_myhostname) ? 1 : 0;
```

Comment:
I don't think `thedb` can be null here. `return (thedb->master == gbl_myhostname);` should be sufficient.